ietf-optical-impairment-topology.tree

```diff
@@ -49,39 +56,110 @@ module: ietf-optical-impairment-topology
                 +--:(concentratedloss)
                    +--ro concentratedloss
                       +--ro loss?   decimal64
-  augment /nw:networks/nw:network/nw:node/tet:te/tet:tunnel-termination-point:
+  augment /nw:networks/nw:network/nw:node/tet:te
+            /tet:tunnel-termination-point:
     +--ro OTSiG-element* [OTSiG-identifier]
     |  +--ro OTSiG-identifier    int16
```

Comment:
This leaf seems not to have any meaning other than being a key in the list.

Do we need a key for this list?

This list is not configuration data (config false), so the key is optional.

If there is no need to reference (via a leaf-ref) an element of this list, this attribute could be removed.

Reply:
> This leaf seems not to have any meaning other than being a key in the list.
> 
> Do we need a key for this list?
> 
> This list is not configuration data (config false), so the key is optional.
> 
> If there is no need to reference (via a leaf-ref) an element of this list, this attribute could be removed.

But this is exactly the case in raw 1142 when you need to get which OTSiG is carried in a media-channel
        leaf OTSiG-ref {
          type leafref {
           path "/nw:networks/nw:network/nw:node/tet:te" +
                "/tet:tunnel-termination-point" +
                "/OTSiG-element/OTSiG-identifier" ;

          }